update/upgrade_db.sh

```diff
@@ -28,25 +35,25 @@ case $key in
     -h|--help)
         echo "Arguments:"
         echo -e "\t-c|--clean\t\tCleans comp and test DB before starting"
-        # echo -e "\t-d|--localdirpath\tRuns a second update cycle with local customization delta files"
+        echo -e "\t-d|--localdirpath\tAdds local customization delta files"
```

Comment:
What if I use both an extension and a customization? Will I be able to use upgrade_db.sh? I understand that upgrade_db.sh supports only one additional extension/customization, but I may be wrong.